tests/components/aurora_abb_powerone/test_config_flow.py

```diff
@@ -150,16 +164,83 @@ async def test_form_invalid_com_ports(hass):
 
 
 # Tests below can be deleted after deprecation period is finished.
-async def test_import(hass):
-    """Test configuration.yaml import used during migration."""
-    TESTDATA = {"device": "/dev/ttyUSB7", "address": 3, "name": "MyAuroraPV"}
+async def test_import_day(hass):
+    """Test .yaml import when the inverter is able to communicate."""
+    TEST_DATA = {"device": "/dev/ttyUSB7", "address": 3, "name": "MyAuroraPV"}
+
+    with patch("aurorapy.client.AuroraSerialClient.connect", return_value=None,), patch(
+        "aurorapy.client.AuroraSerialClient.serial_number",
+        return_value="9876543",
+    ), patch(
+        "aurorapy.client.AuroraSerialClient.version",
+        return_value="9.8.7.6",
+    ), patch(
+        "aurorapy.client.AuroraSerialClient.pn",
+        return_value="A.B.C",
+    ), patch(
+        "aurorapy.client.AuroraSerialClient.firmware",
+        return_value="1.234",
+    ), patch(
+        "homeassistant.components.aurora_abb_powerone.async_setup_entry",
```

Comment:
If we patch the integration setup here we won't do any api calls during import and config entry setup..

Reply:
Agreed. Done.